src/net.c

```diff
@@ -753,6 +760,82 @@ static int hyper_set_interface_mtu(struct rtnl_handle *rth,
 				IFLA_MTU);
 }
 
+static int hyper_get_iface_name_from_mac_addr(char *mac_addr, char **device)
```

Comment:
- Missing checks on pointer parameters.
- `mac_address` can be `const` I think.
- I'd add a function header explaining what this function does, params, return value and also point out that it is limited to handling IPv4 MAC addresses.